codegen/src/main/kotlin/org/web3j/openapi/codegen/utils/GeneratorUtils.kt

```diff
@@ -74,7 +74,9 @@ object GeneratorUtils {
         val distinctInputs = mutableMapOf<String, AbiDefinition.NamedType>()
         inputs.forEachIndexed { index, namedType ->
             if (distinctInputs[argumentName(namedType.name, index).capitalize()] != null) {
-                inputs.filter { input -> input.name == namedType.name }.first().apply {
+                inputs.first { input -> 
+                    input.name == namedType.name 
+                }.apply {
                     this.name = "${this.name}Dup"
```

Comment:
Wasn't this supposed to be an index? What if there are more than 2 duplicates?

Reply:
In the case of `function names` yeah. But this is the case of `inputs`, and there could be the max of two duplicates. We can change the 'Dup' to '2' or something like that, no problem.
For `function names`, I put the `index`.
What do you think ?

Reply:
Is ok leave it like that.